test/check/check.go

```diff
@@ -120,6 +122,36 @@ func PlacementListFromComponentDaemonSets(component Component) ([]cnao.Placement
 	return placementList, nil
 }
 
+func getMonitoringNamespace() (string, error) {
```

Comment:
Why don't we use the production fruncion ?

Reply:
I see no reason why not :)